class PauthError(Exception):
    pass


class OAuthError(PauthError):
    pass
